packages/cdktf-cli/bin/cmds/ui/terraform-context.tsx

```diff
@@ -36,7 +36,7 @@ const parseOutput = (str: string): DeployingResource[] => {
     if (/^Outputs:/.test(line)) { return }
     if (/^data\..*/.test(line)) { return }
 
-    const resourceMatch = line.match(/^([a-zA-Z\d_\-.]*):/)
+    const resourceMatch = line.match(/^([a-zA-Z][a-zA-Z\d_\-.]*):/)
```

Comment:
The spec says the first character isn't a digit, but `-` and `_` may be allowed (could test to confirm).

Reply:
So, like this? https://github.com/hashicorp/terraform-cdk/pull/617/commits/0e7c40d75fad6d77d563775c37112720e0e524d1

Reply:
It actually can't start with a dash.
"A name must start with a letter or underscore and may contain only letters,
digits, underscores, and dashes."

Reply:
Oh, thanks for clarifying :)